modules/store/spec/selector.spec.ts

```diff
@@ -128,7 +128,7 @@ describe('Selectors', () => {
       const selectorFn = jasmine
         .createSpy(
           'selectorFn',
-          createSelector((state) => state, projectorFn)
+          createSelector((state: any) => state, projectorFn)
```

Comment:
i do not know why specifically here TS starting complaining about implicit any, because as far as i can tell this parameter was already implicitly any before this change